generators/respec.js

```diff
@@ -22,17 +22,34 @@ exports.generate = function (url, params, cb) {
             timedout = true;
             cb({ status: 500, message: "Processing timed out." });
         }
-    ,   10000
+    ,   12000
```

Comment:
For large spec like webaudio, I'm getting regular `time out` even with `15000`, but I'm not in the US. How about making it `20s`?
Apart from that, the change LGTM.